src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -138,6 +144,25 @@ function useChatUtilsHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatUtilsHook {
 
   const helperSaveChannelDetail = async (channel: ChannelList, response: any) => {
     if (!localDb) return;
+    response?.messages?.map((message) => {
+      const chatMessage = ChatSchema.fromGetAllChannelAPI(channel?.id, message);
+      return queue.addPriorityJob({
+        label: `saveChatMessage-${message?.id}`,
+        priority: QueueJobPriority.MEDIUM,
+        task: async () => {
+          await chatMessage.save(localDb);
+        }
+      });
+    });
+
+    queue.addPriorityJob({
+      label: 'refresh channel list',
+      priority: QueueJobPriority.MEDIUM,
+      task: async () => {
+        refresh('channelList');
+      }
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the `helperSaveChannelDetail` function, you're using `Array.prototype.map` but not using its return value. If you don't need to create a new array, consider using `Array.prototype.forEach` instead for better readability and performance.
```diff
    response?.messages?.forEach((message) => {
      const chatMessage = ChatSchema.fromGetAllChannelAPI(channel?.id, message);
      queue.addPriorityJob({
        label: `saveChatMessage-${message?.id}`,
        priority: QueueJobPriority.MEDIUM,
        task: async () => {
          await chatMessage.save(localDb);
        }
      });
    });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->